projects/fhi-angular-highcharts/src/lib/models/fhi-diagram-options.model.ts

```diff
@@ -18,6 +18,7 @@ export interface FhiDiagramOptions {
   flags?: Array<FhiDiagramFlag>;
   lastUpdated?: string;
   mapTypeId?: keyof typeof MapTypeIds;
+  hasMetadataLink?: boolean;
```

Comment:
`metadataLinkButton`, more spesific, and no need for has

Reply:
No need for link either... i think `metadataButton` is better, since it can be used to open a modal or whatever...